sds/src/lib.rs

```diff
@@ -38,7 +38,8 @@ pub use path::{Path, PathSegment};
 pub use rule_match::{ReplacementType, RuleMatch};
 pub use scanner::shared_pool::{SharedPool, SharedPoolGuard};
 
-pub use scanner::error::MatchValidationError;
+pub use proximity_keywords::ProximityKeywordsValidationError;
+pub use scanner::error::{MatchValidationError, ScannerMetadataError};
```

Comment:
And these changes would also be required if I still wanted to use the public library's errors